### PR TITLE
 Fix the example for section 09 not to use side effect

### LINKED ISSUE #37: Test for the example in CI
It would be nice if there is a CI tester (such as Travis CI) for the examples in the user guide.
Such a scheme can keep the examples runnable for each commit.

### PR DESCRIPTION
**Pull request by tom-tan**
It partially addresses #37 and #62.

### CODE REVIEW

File: _includes/cwl/09-array-inputs/array-inputs.cwl [hunk 1]
```diff
@@ -27,5 +27,8 @@ inputs:
       separate: false
       position: 4
 
-outputs: []
+outputs:
+  output:
```
Comment (mr-c) [inline _includes/cwl/09-array-inputs/array-inputs.cwl]: Please use a less generic name than `output`; people new to CWL will get confused as it looks similar to the CWL keyword `outputs`

**Comment by mr-c** [comment]
(FYI, I edited the description of this PR; using the word "fixes" would auto-close the mentioned issues when the PR gets merged.)

**Comment by tom-tan** [comment]
> (FYI, I edited the description of this PR; using the word "fixes" would auto-close the mentioned issues when the PR gets merged.)

Thank you for the information. I will use other words from the next time.

File: _includes/cwl/09-array-inputs/array-inputs.cwl (hunk 1 above)
Comment (tom-tan) [reply _includes/cwl/09-array-inputs/array-inputs.cwl]: Thank you for your comment.
I will replace it with `example_out`.

**Pull request merged**